### ISSUE TITLE
which version to use

### ISSUE THREAD

**Issue by lacava** [marcovirgolin/GP-GOMEA#10]
hey, I noticed there are some updates that smooth out the installation process, but I can't build the current master branch version. I get the error:

```
GP-GOMEA/src/gpgomea_python.cpp:220:69: error: no matching function for call to ‘Node::GetSubtreeExpression()’
  220 |             std::string prefix_expression = n->GetSubtreeExpression();
```

srbench is using version 6a92cb671c2772002b60df621a513d8b4df57887 currently. Do you want to update it I run the experiment? let me know.

**Comment by marcovirgolin** [comment]
Thanks for raising this, I need to check why the latest commits did not break that. (@EviSijben I might need your help)
Meanwhile, you can certainly keep using the previous version  ( 6a92cb6 ) because the algorithm works the same way.

**Comment by marcovirgolin** [comment]
My mistake, fixed it with caae139
